sql-parser/statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/statement/ddl/AlterTableStatement.java

```diff
@@ -67,6 +68,8 @@ public abstract class AlterTableStatement extends AbstractSQLStatement implement
     
     private final Collection<DropIndexDefinitionSegment> dropIndexDefinitions = new LinkedList<>();
     
+    private final Collection<RenameIndexDefinitionSegment> renameIndexDefinitions = new LinkedList<>();
```

Comment:
Please add sql parser test case for this pr.

Reply:
[OK]